scipy/optimize/tnc.py

```diff
@@ -152,6 +152,9 @@ def fmin_tnc(func, x0, fprime=None, args=(), approx_grad=0,
     maxfun : int, optional
         Maximum number of function evaluation. If None, maxfun is
         set to max(100, 10*len(x0)). Defaults to None.
+    maxiter : int, optional
```

Comment:
I don't think that we need to add `maxiter` to the keyword arguments.

Reply:
I removed it.